packages/oui/_settings.less

```diff
@@ -15,6 +15,8 @@
   ~".oui-calendar",
   ~".oui-checkbox",
   ~".oui-input",
+  ~".oui-input-group",
```

Comment:
Can't this be generated ou can't you use something like `.oui-formControls` ? So you don't have to update it manually

Reply:
No need to generate this variable. It's a known scope.
Keep it simple